src/edu/csus/ecs/pc2/clics/API202306/CLICSAccount.java

```diff
@@ -0,0 +1,320 @@
+
+    @JsonProperty
+    private String type;
+
+    @JsonProperty
+    private String team_id;
+
+    @JsonProperty
+    private String person_id;
+
+    public CLICSAccount() {
+        // jackson deserialize
+    }
+
+    /**
+     * Fill in properties for an account as per 2023-06 spec
+     *
+     * @param model The contest
+     * @param account team information
+     */
+    public CLICSAccount(IInternalContest model, SecurityContext sc, Account account) {
+        ClientId cid = account.getClientId();
+        id = cid.getName();
+        username = id;
+        name = account.getDisplayName();
+        if(sc.isUserInRole(WebServer.WEBAPI_ROLE_ADMIN)) {
+            password = account.getPassword();
+        }
+        type = "" + cid.getClientType();
+        type = type.toLowerCase();
+        if(account.isTeam()) {
+            team_id = username;
+        }
+    }
+
+    public String toJSON() {
+
+        try {
+            ObjectMapper mapper = JSONUtilities.getObjectMapper();
+            return mapper.writeValueAsString(this);
+        } catch (Exception e) {
+            return "Error creating JSON for account info " + e.getMessage();
+        }
+    }
+
+    /**
+     * Create account list from a accounts.json like file
+     *
+     * @param contest the contest (needed for groups)
+     * @param jsonfile json file to deserialize
+     * @param site the site to create the accounts for
+     * @return array of accounts to add, or null on error
+     */
+    public static Account [] fromJSON(IInternalContest contest, File jsonfile, int site) {
+        Account [] newaccounts = null;
+        Log log = StaticLog.getLog();
+
+        try {
+            ObjectMapper mapper = new ObjectMapper();
+            newaccounts = createAccountsFromCLICSAccounts(contest, mapper.readValue(jsonfile, CLICSAccount[].class), site, log);
+        } catch (Exception e) {
+            log.log(Log.WARNING, "could not deserialize account file " + jsonfile, e);
+        }
+        return(newaccounts);
+    }
+
+    /**
+     * Create account list from a accounts.yaml like file
+     *
+     * @param contest the contest (needed for groups)
+     * @param yamlfile yaml file to deserialize
+     * @param site the site to create the accounts for
+     * @return array of accounts to add, or null on error
+     */
+    public static Account [] fromYAML(IInternalContest contest, File yamlfile, int site) {
+        Account [] newaccounts = null;
+        Log log = StaticLog.getLog();
+
+        try {
+            ObjectMapper mapper = new ObjectMapper(new YAMLFactory());
+            newaccounts = createAccountsFromCLICSAccounts(contest, mapper.readValue(yamlfile, CLICSAccount[].class), site, log);
+        } catch (Exception e) {
+            log.log(Log.WARNING, "could not deserialize account file " + yamlfile, e);
+        }
+        return(newaccounts);
+     }
+
+    /**
+     * Create account list from a string containing accounts yaml
```

Comment:
This comment claims that the method is creating accounts from yaml... but I think the method is actually designed to process (only) JSON.   Same issue with the following comments (they refer to YAML).   Copy/paste error... from Doug?

Reply:
Does it?  I believe the comment says, "Create account list from ..."  
This is for processing JSON configuration files to read in a new contest.  The configuration class that will call this will actually add the accounts from the list returned, presumably.

Reply:
I think you missed my point (or I have missed yours).  I agree with what you said:  `This is for processing JSON configuration files...`.  My point is that the COMMENTS in the method header refer (several times) to processing YAML information (not JSON, which is what the method actually processes).

Reply:
Well... almost.  You missed the part where my original comment also said `Same issue with the following comments (they refer to YAML).`  Specifically, there is an `@param yaml yaml to deserialize` that should also have `s/yaml/json/g` applied.  (I'm actualy surprised that Eclipse doesn't flag that as an error, since the parameter itself is named `json`, not `yaml`...)